analyzers/src/SonarAnalyzer.CSharp/Rules/MethodOverrideNoParams.cs

```diff
@@ -26,32 +26,31 @@ public sealed class MethodOverrideNoParams : SonarDiagnosticAnalyzer
         internal const string DiagnosticId = "S3262";
         private const string MessageFormat = "'params' should not be removed from an override.";
 
-        private static readonly DiagnosticDescriptor rule =
+        private static readonly DiagnosticDescriptor Rule =
             DescriptorFactory.Create(DiagnosticId, MessageFormat);
 
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Rule);
 
-        protected override void Initialize(SonarAnalysisContext context)
-        {
+        protected override void Initialize(SonarAnalysisContext context) =>
             context.RegisterSyntaxNodeActionInNonGenerated(
                 c =>
                 {
                     var method = (MethodDeclarationSyntax)c.Node;
                     var methodSymbol = c.SemanticModel.GetDeclaredSymbol(method);
-                    if (methodSymbol == null ||
```

Comment:
Actually if you invert the conditions you can merge it with the if below which I think should increase the readability. WDYT?